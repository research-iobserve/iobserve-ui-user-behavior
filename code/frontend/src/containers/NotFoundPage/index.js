// @flow

/**
 * NotFoundPage
 *
 * This is the page we show when the user visits a url that doesn't have a route
 */

import React from "react";
import { Container } from "reactstrap";

export default function NotFound() {
  return (
    <Container>
      <h1>
        Page not found.
      </h1>
    </Container>
  );
}
