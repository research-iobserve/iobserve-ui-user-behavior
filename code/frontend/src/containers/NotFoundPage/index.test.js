// @flow

import React from "react";
import renderer from "react-test-renderer";

import NotFoundPage from "./index";

describe("<NotFoundPage />", () => {
  it("should render", () => {
    const tree = renderer.create(<NotFoundPage />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
