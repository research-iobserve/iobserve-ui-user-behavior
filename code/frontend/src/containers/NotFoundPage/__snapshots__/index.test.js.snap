exports[`<NotFoundPage /> should render 1`] = `
<div
  className="container">
  <h1>
    Page not found.
  </h1>
</div>
`;
